Sources/ArgumentParser/Parsable Types/AsyncParsableCommand.swift

```diff
@@ -31,8 +31,17 @@ extension AsyncParsableCommand {
   /// Instead of calling this method directly, you can add `@main` to the root
   /// command for your command-line tool.
   public static func main() async {
+    await self.main(lines: nil)
+  }
+  
+  /// Executes this command, or one of its subcommands, with the program's
+  /// command-line arguments.
+  ///
+  /// Instead of calling this method directly, you can add `@main` to the root
+  /// command for your command-line tool.
+  public static func main(lines: [String]?) async {
```

Comment:
It's a great idea to have a way to test the incremental parsing/initialization that you're adding, but we'll need to find a way to do it without affecting the public `ParsableCommand` API.